cmd/oras/push.go

```diff
@@ -276,22 +276,25 @@ func pushArtifact(dst *remote.Repository, pack packFunc, packOpts *oras.PackOpti
 	return root, nil
 }
 
-func isArtifactUnsupported(err error) bool {
+func skipFallbackToImageManifest(root ocispec.Descriptor, err error) bool {
```

Comment:
Can we have a documentation for this function?

Reply:
Do we still need it now after the refactoring?